addon-test-support/mock-provider/mirage.js

```diff
@@ -55,6 +55,13 @@ export default class MirageProvider extends MockProvider {
       let { startMirage } = require(`${modulePrefix}/initializers/ember-cli-mirage`);
       this.server = startMirage();
       this.server.passthrough('/_pact/*path');
+
+      // Workaround for issue: https://github.com/salsify/ember-cli-pact/issues/32
+      const originalHandler = this.server.pretender.handledRequest;
+      this.server.pretender.handledRequest = (verb, path, request) => {
+        if (verb === 'DELETE') this.recordRequest(request);
+        return originalHandler(verb, path, request);
+      };      
```

Comment:
Cc: @springerigor and @louiseavelar

Reply:
Not an ideal solution. It was inspired on 
https://github.com/salsify/ember-cli-pact/blob/master/addon-test-support/mock-provider/pretender.js#L31